test/userController.test.js

```diff
@@ -93,3 +127,240 @@ describe('GET /api/v1/users/:user_id endpoint', () => {
     expect(response.status).toBe(404);
   });
 });
+
+describe('PATCH /api/v1/users/updateCurrentUser endpoint', () => {
+  it('should update the current user', async () => {
+    // Arrange: Log in and get a signed cookie with valid test user credentials
+    const signedCookie = await loginAndReturnCookie(testUserCredentials);
+
+    const updatedUserData = {
+      email: 'new_email@example.com',
+      name: 'new_name',
+    };
+
+    const response = await request(app)
+      .patch('/api/v1/users/updateCurrentUser')
+      .send(updatedUserData)
+      .set('Cookie', [signedCookie]);
+
+    expect(response.status).toBe(200);
+    expect(response.body).toHaveProperty('user');
+    expect(response.body.user).toMatchObject(updatedUserData);
+  });
+
+  it('should return an error if required data is missing', async () => {
+    const signedCookie = await loginAndReturnCookie({
+      email: 'ava@ava.com',
+      password: 'secret',
+    });
```

Comment:
not a big deal; but to keep things consistent let's use the variable we've already declared with this data:

```suggestion
    const signedCookie = await loginAndReturnCookie(testUserCredentials);
```